### PR TITLE
Update CI and few other things

### PR DESCRIPTION
**Pull request by permanentdaylightt**

### CODE REVIEW

**Review by ScriptedSnark — COMMENTED** [review]
Why is there MinHook reference? I know that original SPTLib uses MinHook but SvenBXT has modified, lite version without that library in favor of FuncHook.

**Review by ScriptedSnark — CHANGES_REQUESTED** [review]
BXT_LICENSE contains only and only BunnymodXT license because of borrowed BXT-specific code. SPTLib already has own license file. `hltas`, `hlstrafe`, `taslogger`,` MinHook` and others are parts of original BunnymodXT, not of SvenBXT.

**Review by ScriptedSnark — COMMENTED** [review]
What is the purpose of these changes in SPTLib-related headers?

**Review by ScriptedSnark — COMMENTED** [review]
I think .gitignore can be extended with lots of Visual Studio/IDEA/CMake temporary files/formats/binaries.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -35,7 +34,7 @@ jobs:
             bin-path: "src/build/Release/SvenBXT.dll"
             toolchain-file: ""
             cmake-generator: "Visual Studio 17 2022"
-            build-target: "-A Win32" # HACK, but I have no idea how to do it
```
Comment (keyboardcrash1) [inline .github/workflows/CI.yml:38]: What is the purpose of deleting that comment?

**Comment by permanentdaylightt** [comment]
Well yeah youre right I might made some unrequired changes

**Pull request closed without merging**